docs/sources/configure-client/trace-span-profiles/dotnet-span-profiles.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Span profiles with Traces to profiles for .NET
+menuTitle: Span profiles with Traces to profiles (.NET)
+description: Learn about and configure Span profiles with Traces to profiles in Grafana for .NET applications.
+weight: 103
+---
+
+# Span profiles with Traces to profiles for .NET
+
+![span-profiles screenshot](https://grafana.com/static/img/docs/tempo/profiles/tempo-profiles-Span-link-profile-data-source.png)
+
+## Before you begin
+
+Your applications must be instrumented for profiling and tracing before you can use span profiles.
+
+* Profiling: Your application must be instrumented with Pyroscope's .NET instrumentation library. If you haven't done this yet, please refer to the [.NET]({{< relref "../language-sdks/dotnet" >}}) guide.
+* Tracing: Your application must be instrumented with OpenTelemetry traces. If you haven't done this yet, please refer to the [OpenTelemetry](https://opentelemetry.io/docs/net/getting-started/) guide.
+
+{{< admonition type="note" >}}
+Currently only [OpenTelemetry manual instrumentation](https://opentelemetry.io/docs/languages/net/instrumentation/) is supported for span profiles in .NET.
+This limitation is there because Pyroscope's .NET profiler and OpenTelemetry's auto instrumentation are based on separate .NET CLR profilers.
+{{< /admonition >}}
+
+## OpenTelemetry support
```

Comment:
This section is really good. What would you think about using this as part of the introduction? This sections provides the explanation about OTel support, which is also highlighted in the Before you begin section. Moving the Otel Support into the introduction would immediately point out the limitations at the top of the page. 

If the other integrations for span profiles have the same limitation with OTel, then I'd recommend using the same section and introduction in each of those pages as well.

Reply:
I updated the intro for this page above to include this section.  if you use that suggestions, make sure and delete the OpenTelemetry section here to avoid duplication.